kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -43,7 +43,12 @@ grCollectionsPanel.controller('GrNodeCtrl',
     const ctrl = this;
     ctrl.saving = false;
     ctrl.deletable = false;
-    ctrl.addChild = childName => collections.addChildTo(ctrl.node, childName);
+    ctrl.formError = null;
+    ctrl.addChild = childName =>
```

Comment:
Maybe it's just me, but I would reserve `{}`-free arrow functions to one-liners. Possibly reminds me too much of Coffee:hankey:Script :mask: